README.md

```diff
@@ -24,6 +26,12 @@ Remember that this is a collaborative content, so, if something is missing or in
 
 ![Android Roadmap](./images/android_roadmap.png)
 
+
+
+## Android app
+
+To help you keep track of your progress we created an Android app with all the subjects. You can download the app [here](https://play.google.com/store/apps/details?id=com.fren.androiddeveloperroadmap). 
```

Comment:
@Fren1100 how about linking your Github account here? Since wasn't us (co-criators) that created, would be nice to tag you here as a contributor 😊  (and the other README files)